src/app/helptext/services/components/service-smart.ts

```diff
@@ -11,6 +11,17 @@ export default {
  tests are configured to run.'),
   smart_interval_validation: [Validators.required],
 
+  smart_powermode_placeholder: T('Power Mode'),
+  smart_powermode_tooltip: T('Tests are only performed when <i>Never</i> \
+is selected.'),
+  smart_powermode_options: [
+    { label: 'Never', value: 'NEVER' },
+    { label: 'Sleep', value: 'SLEEP' },
+    { label: 'Standby', value: 'STANDBY' },
+    { label: 'Idle', value: 'IDLE' },
+  ],
+  smart_powermode_validation: [Validators.required],
```

Comment:
I think helptext files are only intended for text hints, logic belongs back to component.